launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,74 @@
+import io
+import re
+from launch.output_handler import LineOutput
```

Comment:
Yes, I ran it locally. Not in isolation. And thanks for keeping an eye on this, but I was already working on a fix.